servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1795,6 +1819,141 @@ private void setStateFromEntry(SetStateEntry entry) throws FileDoesNotExistExcep
   }
 
   /**
+   * Sets the acl of a file or directory. Only one of owner, group, or permission in the
```

Comment:
but this seems an artificially imposed limitation and I could imagine having an setacl command later down the road that let's you set all three in one command (e.g. for efficiency) ... would the implementation not do the right thing if multiple attributes were set?